plugin/src/main/java/org/opensearch/ml/sdkclient/DDBOpenSearchClient.java

```diff
@@ -321,14 +352,36 @@ private XContentParser createParser(String json) throws IOException {
         return jsonXContent.createParser(NamedXContentRegistry.EMPTY, DeprecationHandler.IGNORE_DEPRECATIONS, json);
     }
 
-    private String simulateOpenSearchResponse(String index, String id, String source, Map<String, Object> additionalFields) {
+    private GetItemRequest getGetItemRequest(String requestTenantId, String documentId, String index) {
```

Comment:
(Style) `getGet...` looks like a typo even if it's not. How about `buildGet...` ?